docs/victoriametrics/Cluster-VictoriaMetrics.md

```diff
@@ -462,6 +462,15 @@ for sending data from `vminsert` to `vmstorage` node according to `-vminsertAddr
 
 The currently discovered `vmstorage` nodes can be [monitored](#monitoring) with `vm_rpc_vmstorage_is_reachable` and `vm_rpc_vmstorage_is_read_only` metrics.
 
+The discovered addresses preserve [vmstorage group](#vmstorage-groups-at-vmselect) prefix:
```

Comment:
Discovery could be used together (or combined) with [groups](#vmstorage-groups-at-vmselect). See example: